php/commands/site.php

```diff
@@ -1607,32 +1584,42 @@ public function wipe($args, $assoc_args) {
     Terminus::success(sprintf("Successfully wiped %s - %s", $site->getName(), $environment_id));
   }
 
-   /**
-    * Delete a site from pantheon
-    *
-    * ## OPTIONS
-    * [--site=<site>]
-    * : ID of the site you want to delete
-    *
-    * [--force]
-    * : to skip the confirmations
-    */
-   function delete($args, $assoc_args) {
-     $sitename = Input::sitename($assoc_args);
-     $site_id = $this->sitesCache->findID($sitename);
-     $site_to_delete = new Site($site_id);
-
-     if (!isset($assoc_args['force']) AND !Terminus::get_config('yes')) {
-       // if the force option isn't used we'll ask you some annoying questions
-       Terminus::confirm( sprintf( "Are you sure you want to delete %s?", $site_to_delete->information->name ));
-       Terminus::confirm( "Are you really sure?" );
-     }
-     Terminus::line( sprintf( "Deleting %s ...", $site_to_delete->information->name ) );
-     $response = \TerminusCommand::request( 'sites', $site_to_delete->id, '', 'DELETE' );
-
-     $this->sitesCache->remove($sitename);
-     Terminus::success("Deleted %s!", $sitename);
-   }
+  /**
+   * List a site's workflows
+   *
+   * ## OPTIONS
+   * [--site=<site>]
+   * : Site to check
+   *
+   * @subcommand workflows
+   */
+  public function workflows($args, $assoc_args) {
+    $site = SiteFactory::instance(Input::sitename($assoc_args));
+    $workflows = $site->getWorkflows();
+    $data = array();
+    foreach($workflows as $workflow) {
+      $user = 'Pantheon';
```

Comment:
It would be nice if the API handled this so that the client didn't have to munge this field. (Though, that shouldn't block this PR.)